cmd/edgectl/aes_install.go

```diff
@@ -437,166 +429,100 @@ func (i *Installer) Perform(kcontext string) error {
 		return err
 	}
 
-	// Allow overriding the source domain (e.g., for smoke tests before release)
-	manifestsDomain := "www.getambassador.io"
-	domainOverrideVar := "AES_MANIFESTS_DOMAIN"
-	overrideMessage := "Downloading manifests from override domain %q instead of default %q because the environment variable %s is set."
-	if amd := os.Getenv(domainOverrideVar); amd != "" {
-		i.ShowWrapped(fmt.Sprintf(overrideMessage, amd, manifestsDomain, domainOverrideVar))
-		manifestsDomain = amd
+	namespace, err := i.kubeinfo.Namespace()
```

Comment:
It would be safer to assume we are installing AES in the `ambassador` namespace (and create the namespace if necessary). 

Although I agree it would be great to allow customizing the namespace and allow installs anywhere, a lot of the installer code assumes we are in the `ambassador` namespace for now. I would feel more confident to stick with a hardcoded namespace for now and refactor in another PR.

Reply:
Then shouldn't we set explictly the `ambassador` namespace in [`i.kubeinfo = k8s.NewKubeInfo(...)`](https://github.com/datawire/ambassador/blob/master/cmd/edgectl/aes_install.go#L379)?

Reply:
Ok, I have defined

```go
	// defInstallNamespace is the default installation namespace
	defInstallNamespace = "ambassador"
```

and I will use `defInstallNamespace` instead of this `namespace`.